### ISSUE TITLE
Failed to retrieve information ... | Add verbose mode?

### ISSUE THREAD

**Issue by 0x53A** [Zaid-Ajaj/Femto#70]
![image](https://user-images.githubusercontent.com/4236651/88207047-7d9ac880-cc4f-11ea-91ec-6ea2496daeb0.png)

**Comment by 0x53A** [comment]
Could it be the warning?

![image](https://user-images.githubusercontent.com/4236651/88207193-b9ce2900-cc4f-11ea-9227-4a40de1b838d.png)

**Comment by Zaid-Ajaj** [comment]
> Could it be the warning?

Yeah I believe so, maybe we should fallback to npm when yarn fails 🤔 (PR would be awesome!)

**Comment by Zaid-Ajaj** [comment]
The problem should be coming somewhere from [here](https://github.com/Zaid-Ajaj/Femto/blob/master/src/Program.fs#L189-L201)

**Comment by 0x53A** [comment]
I think the warning is printed to STDERR, so the check ``String.isNullOrEmpty res.Result.Error`` will fail ...

I could just skip lines that start with ``(node:#number#) Warning``?
